api/tests/unit/organisations/test_unit_organisations_models.py

```diff
@@ -85,7 +85,8 @@ def test_cancel_subscription_cancels_chargebee_subscription(
         )
         # refresh subscription object
         subscription.refresh_from_db()
-        assert subscription.cancellation_date
+        # Subscription has been immediately transformed to free.
+        assert subscription.cancellation_date is None
```

Comment:
Feels like there should be more asserts here?

Reply:
This test was originally targeted at chargebee and other tests enforced the other checks, but I've added more asserts because there is no downside.